draft-ietf-webpush-protocol.xml

```diff
@@ -1084,7 +1084,7 @@ HEADERS      [stream 82] +END_STREAM
             A push service MAY expire a subscription set at any time which MUST also
             expire all push message subscriptions in the set. If a user agent has an
             outstanding request to a push subscription set (<xref target="monitor-set"/>)
-            this can be signaled by returning a 400-series status code, such as 410 (Gone).
+            this can be signaled by returning a 410 (Gone) status code.

```

Comment:
It's specific to gateway or proxy timeouts though:

 _The 504 (Gateway Timeout) status code indicates that the server, **while acting as a gateway or proxy**, did not receive a timely response from an upstream server it needed to access in order to complete the request._